src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -708,7 +713,8 @@ public static int Compare(BigInteger left, BigInteger right)
 
         public static BigInteger Abs(BigInteger value)
         {
-            return (value >= Zero) ? value : -value;
+            value.AssertValid();
+            return new BigInteger(unchecked((int)NumericsHelpers.Abs(value._sign)), value._bits, true);
```

Comment:
AssertValid makes sure _sign is not int.MinValue, so Math.Abs can't throw

```suggestion
            return new BigInteger(Math.Abs(value._sign), value._bits, true);
```

Reply:
`_sign` cannot be `int.MinValue` if the `BigInteger` is valid. Afaik operations on an invalid `BigInteger` is UB.

https://github.com/dotnet/runtime/blob/658b4b8041888f7e277da075a25dcfb9fd7605d8/src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs#L39-L41

Besides `AssertValid()` executes only in debug versions of CLR.